euclidean_gcd/CPP/euclidean_gcd.cpp

```diff
@@ -0,0 +1,24 @@
+#include <bits/stdc++.h>
+#define ll long long
+using namespace std;
+
+ll gcd(ll a, ll b)
+{
+    if (a == 0)
+        return b;
+    if (b == 0)
+        return a;
+    return gcd(b % a, a);
+}
+
+int main()
+{
+    // a and b are two numbers.
+    ll a, b;
+    cin >> a >> b;
+
+    ll ans = gcd(a, b);
+    cout << ans;
+
+    return 0;
+}
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `all.space_checks`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpt10e9fqt/euclidean_gcd/CPP/euclidean_gcd.cpp
+++ b/tmp/tmpt10e9fqt/euclidean_gcd/CPP/euclidean_gcd.cpp
@@ -21,4 +21,4 @@
     cout << ans;
 
     return 0;
-}+}
```